qemu/tests/cfg/vdpa_dpdk.cfg

```diff
@@ -0,0 +1,41 @@
+- vdpa_dpdk:
+    no JeOS
+    no Windows
```

Comment:
Seems missing `only q35` here.

Reply:
I have confirmed with developer , vdpa support with/without q35, so I remove the q35.

Reply:
OK, as RHEL only x86 supports vdpa, but other users can rebuild kernel to support it, so let's keep your current code.
Could you provide latest result?